src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/stores/DatagridStore.js

```diff
@@ -133,7 +157,9 @@ export default class DatagridStore {
     }
 
     destroy() {
-        this.disposer();
+        if (this.disposer) {
```

Comment:
This probably also wouldn't be necessary if we let the `autorun` in the `constructor`